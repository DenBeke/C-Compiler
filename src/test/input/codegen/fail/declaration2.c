// Undeclared...

int main() {

    a = 5;

}