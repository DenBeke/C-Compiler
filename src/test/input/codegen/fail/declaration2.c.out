[31m[ERROR] line 5: Use of undeclared 'a'[0m
